zookeeper-assembly/src/main/assembly/bin-package.xml

```diff
@@ -0,0 +1,129 @@
+<!--
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+-->
+  <id>bin-package</id>
+  <formats>
+    <format>tar.gz</format>
+    <format>zip</format>
+  </formats>
+  <includeBaseDirectory>false</includeBaseDirectory>
+  <componentDescriptors>
+    <componentDescriptor>src/main/assembly/components.xml</componentDescriptor>
+  </componentDescriptors>
+
+  <moduleSets>
+    <!-- ZooKeeper jars (excluding pom projects) including 3rd party dependencies -->
+    <moduleSet>
+      <useAllReactorProjects>true</useAllReactorProjects>
+      <includes>
+        <include>org.apache.zookeeper:zookeeper-server</include>
+        <include>org.apache.zookeeper:zookeeper-recipes-election</include>
+        <include>org.apache.zookeeper:zookeeper-recipes-lock</include>
+        <include>org.apache.zookeeper:zookeeper-recipes-queue</include>
+      </includes>
+
+      <binaries>
+        <outputDirectory>lib</outputDirectory>
+        <unpack>false</unpack>
+        <dependencySets>
+          <dependencySet>
+            <excludes>
+              <exclude>org.apache.zookeeper:zookeeper-recipes</exclude>
+              <exclude>org.apache.zookeeper:zookeeper-client</exclude>
+              <exclude>org.apache.zookeeper:zookeeper-docs</exclude>
+            </excludes>
+          </dependencySet>
+        </dependencySets>
+      </binaries>
+    </moduleSet>
+  </moduleSets>
+
+  <fileSets>
+    <fileSet>
+      <!-- ZooKeeper generated documents -->
+      <directory>${project.basedir}/../zookeeper-docs/target/html</directory>
+      <outputDirectory>docs</outputDirectory>
+      <fileMode>${rw.file.permission}</fileMode>
+      <directoryMode>${rwx.file.permission}</directoryMode>
+    </fileSet>
+    <fileSet>
+      <!-- License files for 3rd party libs -->
+      <directory>${project.basedir}/../zookeeper-server/src/main/resources/lib</directory>
+      <includes>
+        <include>*.txt</include>
+      </includes>
+      <outputDirectory>lib</outputDirectory>
+      <fileMode>${rw.file.permission}</fileMode>
+    </fileSet>
+
+    <!-- Add source jars "manually", as the :source classiffier doesn't work in moduleSets -->
```

Comment:
Do you have a link to a JIRA on the maven-assembly-plugin  ?
https://issues.apache.org/jira/projects/MASSEMBLY

Reply:
I will create one, thanks!